### PR TITLE
IOTEVENT-467: update to latest dropwizard version, update to latest c…

### PR DESCRIPTION
**Pull request by edwardsoo-ss**
…assandra driver version for 2.1

@jeff-blaisdell @Bijnagte have a look please

### CODE REVIEW

**Comment by edwardsoo-ss** [comment]
published and tested `dropwizard-cassandra` with ticker locally

**Comment by Bijnagte** [comment]
you should also change https://github.com/SmartThingsOSS/dropwizard-common/blob/master/version.txt to keep it in line with the dropwizard version

**Comment by jeff-blaisdell** [comment]
Other than Dylan's comments rest looks good to me.

**Comment by edwardsoo-ss** [comment]
changed version.txt to 1.0.5-SNAPSHOT

File: version.txt [hunk 1]
```diff
@@ -1 +1 @@
-1.0.3.4-SNAPSHOT
+1.0.5-SNAPSHOT
```
Comment (jeff-blaisdell) [inline version.txt]: This should be `1.0.5.0-SNAPSHOT`

The pattern we've been using is ${dropwizard-veresion}-${increment}

**Comment by Bijnagte** [comment]
👍

**Comment by jeff-blaisdell** [comment]
👍

**Comment by edwardsoo-ss** [comment]
thanks

**Pull request merged**